docs/wiki/99_Developers/Developers_Guide.md

```diff
@@ -219,10 +220,10 @@ You may ask for someone to review your PR at your discretion. However, do not as
 In general, if someone has previously reviewed your PR and requested changes, it is good practice to re-request their review once you have implemented those changes.
 
 If you have requested review on a PR and have not received feedback for a number of days, it is okay to either re-request their review or leave a comment
-on the PR pinging them with a short message. For example, "Hey @developer, could you please review this PR and let me know what you think!"
+on the PR pinging the developer with a short message. For example, "Hey @developer, could you please review this PR and let me know what you think!"
 
 #### Implementing suggestions or requested changes
-It is good practice to implement any requested changes as soon as possible and then re-request review from the developer who requested them. It is also recommended
+It is good practice to implement any requested changes as soon as possible and then re-request review from the developer. It is also recommended
```

Comment:
```suggestion
It is good practice to implement any requested changes as soon as possible and then re-request review from the developer. It is also helps the team if you acknowledge each comment/request with a few words or just an emoji, so that anyone reading will know you've seen it.```